src/tokens/ERC5095.sol

```diff
@@ -239,7 +341,7 @@ contract ERC5095 is ERC20Permit, IERC5095 {
             underlying,
             maturity,
             assets,
-            Cast.u128(s - (s / 100))
```

Comment:
When i mean 0 slippage i didnt mean set slippage to `int(0)` but to ensure there is 0 slippage. 

So in context here it would mean inputting `shares` as the minimum amount of PTs to receive when spending the calculated `assets`.